src/info/manifest.c

```diff
@@ -0,0 +1,1986 @@
+    STRING_STATIC(MANIFEST_KEY_BACKUP_PRIOR_STR,                    MANIFEST_KEY_BACKUP_PRIOR);
+#define MANIFEST_KEY_BACKUP_TIMESTAMP_COPY_START                    "backup-timestamp-copy-start"
+    STRING_STATIC(MANIFEST_KEY_BACKUP_TIMESTAMP_COPY_START_STR,     MANIFEST_KEY_BACKUP_TIMESTAMP_COPY_START);
+#define MANIFEST_KEY_BACKUP_TIMESTAMP_START                         "backup-timestamp-start"
+    STRING_STATIC(MANIFEST_KEY_BACKUP_TIMESTAMP_START_STR,          MANIFEST_KEY_BACKUP_TIMESTAMP_START);
+#define MANIFEST_KEY_BACKUP_TIMESTAMP_STOP                          "backup-timestamp-stop"
+    STRING_STATIC(MANIFEST_KEY_BACKUP_TIMESTAMP_STOP_STR,           MANIFEST_KEY_BACKUP_TIMESTAMP_STOP);
+#define MANIFEST_KEY_BACKUP_TYPE                                    "backup-type"
+    STRING_STATIC(MANIFEST_KEY_BACKUP_TYPE_STR,                     MANIFEST_KEY_BACKUP_TYPE);
+#define MANIFEST_KEY_CHECKSUM                                       "checksum"
+    VARIANT_STRDEF_STATIC(MANIFEST_KEY_CHECKSUM_VAR,                MANIFEST_KEY_CHECKSUM);
+#define MANIFEST_KEY_CHECKSUM_PAGE                                  "checksum-page"
+    VARIANT_STRDEF_STATIC(MANIFEST_KEY_CHECKSUM_PAGE_VAR,           MANIFEST_KEY_CHECKSUM_PAGE);
+#define MANIFEST_KEY_CHECKSUM_PAGE_ERROR                            "checksum-page-error"
+    VARIANT_STRDEF_STATIC(MANIFEST_KEY_CHECKSUM_PAGE_ERROR_VAR,     MANIFEST_KEY_CHECKSUM_PAGE_ERROR);
+#define MANIFEST_KEY_DB_ID                                          "db-id"
+    STRING_STATIC(MANIFEST_KEY_DB_ID_STR,                           MANIFEST_KEY_DB_ID);
+    VARIANT_STRDEF_STATIC(MANIFEST_KEY_DB_ID_VAR,                   MANIFEST_KEY_DB_ID);
+#define MANIFEST_KEY_DB_LAST_SYSTEM_ID                              "db-last-system-id"
+    VARIANT_STRDEF_STATIC(MANIFEST_KEY_DB_LAST_SYSTEM_ID_VAR,       MANIFEST_KEY_DB_LAST_SYSTEM_ID);
+#define MANIFEST_KEY_DB_SYSTEM_ID                                   "db-system-id"
+    STRING_STATIC(MANIFEST_KEY_DB_SYSTEM_ID_STR,                    MANIFEST_KEY_DB_SYSTEM_ID);
+#define MANIFEST_KEY_DB_VERSION                                     "db-version"
+    STRING_STATIC(MANIFEST_KEY_DB_VERSION_STR,                      MANIFEST_KEY_DB_VERSION);
+#define MANIFEST_KEY_DESTINATION                                    "destination"
+    VARIANT_STRDEF_STATIC(MANIFEST_KEY_DESTINATION_VAR,             MANIFEST_KEY_DESTINATION);
+#define MANIFEST_KEY_FILE                                           "file"
+    VARIANT_STRDEF_STATIC(MANIFEST_KEY_FILE_VAR,                    MANIFEST_KEY_FILE);
+#define MANIFEST_KEY_GROUP                                          "group"
+    STRING_STATIC(MANIFEST_KEY_GROUP_STR,                           MANIFEST_KEY_GROUP);
+    VARIANT_STRDEF_STATIC(MANIFEST_KEY_GROUP_VAR,                   MANIFEST_KEY_GROUP);
+#define MANIFEST_KEY_PRIMARY                                        "ma" "st" "er"
```

Comment:
I've been trying to expunge the word "master" from the repo and I don't want this to come up in searches.  We can't change it, of course, because it would break the repo format.